test/ir_Zepeal_test.cpp

```diff
@@ -0,0 +1,312 @@
+// Copyright 2020 Christian Nilsson
+
+#include "IRac.h"
+#include "IRrecv.h"
+#include "IRrecv_test.h"
+#include "IRsend.h"
+#include "IRsend_test.h"
+#include "gtest/gtest.h"
+
+// Tests for decodeZepeal().
+
+TEST(TestDecodeZepeal, RealExample) {
+  IRsendTest irsend(kGpioUnused);
+  IRrecv irrecv(kGpioUnused);
+  // fuuryou (speed) one of 5 repeats
+  const uint16_t rawData_1[35] = {
+      2328, 3412,
+      424, 1314,  1302, 436,  1276, 454,  424, 1316,
+      1274, 464,  1274, 458,  454, 1278,  450, 1288,
+      1302, 432,  424, 1306,  424, 1306,  424, 1306,
+      428, 1304,  450, 1290,  1298, 430,  426, 1308,
+      426};  // UNKNOWN B5E66F84
+  irsend.begin();
+  irsend.reset();
+  irsend.sendRaw(rawData_1, 35, 38);
+  irsend.makeDecodeResult();
+
+  ASSERT_TRUE(irrecv.decodeZepeal(&irsend.capture));
+  ASSERT_EQ(decode_type_t::ZEPEAL, irsend.capture.decode_type);
+  ASSERT_EQ(kZepealBits, irsend.capture.bits);
+  EXPECT_EQ(0x6C82, irsend.capture.value);
```

Comment:
If there are only a few buttons/values on the remote, maybe consider adding them as constants or creating a comment section in ir_Zepeal.cpp listing them all. Your call if you want to do ANY of that, completely optional/bonus points etc. :-)